mojo/core/ports/node.cc

```diff
@@ -876,9 +880,8 @@ int Node::OnObserveProxy(const PortRef& port_ref,
     }
   }
 
-  if (event_to_forward) {
+  if (event_to_forward)
     delegate_->ForwardEvent(event_target_node, std::move(event_to_forward));
-  }
```

Comment:
why this change?  format-on-save?

Reply:
This was not in the vanilla version. We added it at some point, so I removed it to reduce conflicts 🤷‍♀️